NEWS.md

```diff
@@ -1,5 +1,10 @@
 # vctrs 0.1.0.9000
 
+* New `vec_proxy()` generic. This is the main customisation point in
```

Comment:
Do you have an example where someone would need to implement this? Like does it change anything in the S3 vector? Or is this primarily needed to allow vctrs to with existing vector classes? In that case, can you point to an example where you'd need to define a `vec_proxy()` method?

Reply:
Should we implement `vec_proxy.POSIXlt()` as an example?

Reply:
With the new scheme we discussed on Slack (records return data frames), `vec_proxy.POSIXlt()` now returns a data frame.

I have updated the documentation in NEWS and `?vec_proxy`.